biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -397,7 +399,7 @@
 	String							COMPONENT_MULTIPLE							= "multiple:";
 	String							COMPONENT_GREEDY							= "greedy:";
 	String							COMPONENT_PROVIDE							= "provide:";
-	String							COMPONENT_OPTIONAL							= "optional:";
+	String							COMPONENT_OPTIONAL							= OPTIONAL + ":";
```

Comment:
Not sure this is a good coupling. While they both share the same root word, it is not clear their is any relationship here.